### PR TITLE
Subtheming and usability tweaks for Polaris/Map Menu

### PR DESCRIPTION
**Pull request by winstonwumbo**
## New Features
* Support for overloading render methods has been added to `polaris-flex-theme`
* `polaris-flex-theme` can be extended into subthemes
* Responsive width added to content container, allowing visibility when editor menu is open
* Refactored the CSS for various slots and render methods to support dropping in `site-region` content
## Related Issue(s)
* haxtheweb/issues#2023
* haxtheweb/issues/1199

### CODE REVIEW

File: elements/a11y-collapse/a11y-collapse.js
Comment (btopro) [inline elements/a11y-collapse/a11y-collapse.js:1]: make sure the update is also in `/src/` file name

File: elements/map-menu/map-menu.js
Comment (btopro) [inline elements/map-menu/map-menu.js:1]: same here. make sure `/src/` version also gets the update

File: elements/a11y-collapse/a11y-collapse.js
Comment (winstonwumbo) [reply elements/a11y-collapse/a11y-collapse.js:1]: Done

File: elements/map-menu/map-menu.js
Comment (winstonwumbo) [reply elements/map-menu/map-menu.js:1]: Done

**Pull request merged**